src/main/java/it/gov/pagopa/pu/pagopapayments/event/producer/RegistryProducerService.java

```diff
@@ -0,0 +1,117 @@
+package it.gov.pagopa.pu.pagopapayments.event.producer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.pu.pagopapayments.enums.RegistryEventCategory;
+import it.gov.pagopa.pu.pagopapayments.enums.RegistryEventOutcome;
+import it.gov.pagopa.pu.pagopapayments.enums.RegistryEventSubType;
+import it.gov.pagopa.pu.pagopapayments.enums.RegistryEventType;
+import it.gov.pagopa.pu.pagopapayments.event.producer.dto.RegistryEventDTO;
+import it.gov.pagopa.pu.pagopapayments.exception.ApplicationException;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.kafka.support.KafkaHeaders;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Component;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import java.util.function.Supplier;
+
+@Component
+@Slf4j
+public class RegistryProducerService {
+  public static final String PU_ID = "piattaformaunitaria";
+  public static final String NODE_ID = "NodoDeiPagamentiSPC";
```

Comment:
let' move these constants on RegistryLogger class.

ps: I'm fixing this also in pu-sil